pages/configuration/advanced.md

```diff
@@ -62,6 +62,29 @@ By default we assign 16MB of memory to the GPU (specified by `gpu_mem`.) This
 may well be less than you require depending on your application (particularly
 applications which make heavy use of the Pi's graphics card.)
 
+### Enable serial interface
+
+The BCM2837 on the Raspberry Pi3 has 2 UARTs (as did its predecessors), however to support the Bluetooth functionality the fully featured PL011 UART was moved from the header pins to the Bluetooth chip and the mini UART made available on header pins 8 & 10.
+
+This has a number of consequences for users of the serial interface.
+ - The /dev/ttyAMA0 previously used to access the UART now connects to Bluetooth.
```

Comment:
If we are documenting already, would it worth adding info about `RESIN_HOST_CONFIG_dtoverlay` setting to `pi3-miniuart-bt-overlay`, so one can use the `/dev/ttyAMA0` serial to communicate on the GPIO pins??